websites/S/Super Mangás/dist/metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+  "author": {
+    "name": "Nanashin",
+    "id": "330030557457022986"
+  },
+  "service": "Super Mangás",
+  "description": {
+    "en": "Read manga in Super Mangas easily on any platform, whether mobile or not.",
+    "pt_BR": "Leia mangás no Super Mangás facilmente em qualquer plataforma, mobile ou não.",
+    "pt": "Leia mangás no Super Mangás facilmente em qualquer plataforma, mobile ou não."
+  },
+  "url": "www.supermangas.site",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/uGXva29.png",

```

Comment:
Can you change the logo with one that has better quality?

Reply:
The quality of the presence logo is the same as that of the SA presence logo. It would be better to leave it at that since both are the same quality so as not to cause strangeness.

Reply:
Even though the image provided is on 512x, the logo was not properly upscaled, therefore what you provided is considered as a low-quality image.

If the website doesn't provide an higher-quality version of the logo, we recommend [waifu2x](http://waifu2x.udp.jp/), even though it's not always 100% perfect, it's better than a pixelated image.

Reply:
Logo of the SA presence in the website: 
![image](https://user-images.githubusercontent.com/63855357/85971524-c12d4a00-b9a3-11ea-90ed-7a6a015c954f.png)
Logo of the SM in website (imgur link in src changed)
![image](https://user-images.githubusercontent.com/63855357/85971580-e91cad80-b9a3-11ea-9e75-067c955370b7.png)
Logo in premid menu:
![image](https://user-images.githubusercontent.com/63855357/85971646-1ff2c380-b9a4-11ea-9bb8-b62fc350e394.png)
Logo in the rich presence:
![image](https://user-images.githubusercontent.com/63855357/85971680-31d46680-b9a4-11ea-86dc-a219579e47fc.png)

Reply:
Seems fine.